File: Ass1.py
import random
import matplotlib.pyplot as lala
import pandas as pd
import numpy as freak
from scipy import stats
	
series = pd.Series(freak.random.randint(0,101,1000000))
print('1,000,000 Data-\n****************\nMean: ',series.mean(),'\nMedian: ',series.median(),'\nMode: ',stats.mode(series))
fig = lala.figure()
lala.hist(series)
fig.savefig('hist_full')

sample1 = series.sample(n = 10000)
print('\n10,000 Data-\n****************\nMean: ',sample1.mean(),'\nMedian: ',sample1.median(),'\nMode: ',stats.mode(sample1))
fig2 = lala.figure()
lala.hist(sample1)
fig2.savefig('hist_sample1')

count=0
i=0
halla=[]
while count<10000 :
	halla.append(series[i])
	count+=1
	i=i+5	

sample2 = pd.Series(halla)
print('\n10,000 step-sized Data-\n****************\nMean: ',sample2.mean(),'\nMedian: ',sample2.median(),'\nMode: ',stats.mode(sample2))
fig3 = lala.figure()
lala.hist(sample2)
fig3.savefig('hist_sample2')

ser = series.tolist()
values = [ser,sample1,sample2]
fig1 = lala.figure()
lala.boxplot(values)
fig1.savefig('box')

File: Ass1.r
chup <- data.frame(y = sample(0:100,1000000,replace=T))
chup.mean <- mean(chup$y)
chup.median <- median(chup$y)
chup.mode <- names(table(chup$y))[table(chup$y)==max(table(chup$y))]
cat("1,000,000 data:\n****************\n","Mean: ",chup.mean,"\nMedian: ",chup.median,"\nMode: ",chup.mode,fill=TRUE)
sample1 <- sample(chup$y,10000,replace=F)
nikal.aaja <- names(table(sample1)[table(sample1)==max(table(sample1))])
cat("10,000 data:\n****************\n","Mean: ",mean(sample1),"\nMedian: ",median(sample1),"\nMode: ",nikal.aaja,fill=TRUE)
l <- list()
ginti <- 10000
i <- 1
while (ginti>0) {
  l <- c(l,chup$y[i])
  i =i+5
  ginti = ginti-1
}
kaka <- unlist(l)
nikal.jaa <- names(table(kaka)[table(kaka)==max(table(kaka))])
cat("10,000 step-sized data:\n****************\n","Mean: ",mean(kaka),"\nMedian: ",median(kaka),"Mode: ",nikal.jaa,fill=TRUE)
plot(chup)
hist(chup$y)
